packages/ui-components/src/components/button.tsx

```diff
@@ -28,18 +33,13 @@ export interface IButtonProps
  * @returns Component
  */
 export function Button(props: IButtonProps): JSX.Element {
-  const { minimal, small, children, ...others } = props;
+  const { small, children, className, ...others } = props;
   return (
-    <button
+    <ToolkitButton
       {...others}
-      className={classes(
-        props.className,
-        minimal ? 'jp-mod-minimal' : '',
-        small ? 'jp-mod-small' : '',
-        'jp-Button'
-      )}
+      className={classes(className, small ? 'jp-mod-small' : '', 'jp-Button')}
```

Comment:
Let's not change that as consumer of the Button may use those classes to tune the styling of the button.

Reply:
I just removed the minimal here as the toolkit button has the minimal attribute so i used the toolkit button's minimal attribute if that is not expected i will change to the old class.